package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@azure/functions",
-    "version": "4.7.3",
+    "version": "4.7.4",
```

Comment:
This is just reverting the code we do not need to update the version, can you revert to 4.7.2.